crates/cxx-qt-gen/test_inputs/inheritance.rs

```diff
@@ -9,6 +9,11 @@ mod inheritance {
         type QAbstractItemModel;
     }
 
+    extern "C++Qt" {
+        #[qobject]
+        type MyStruct;
+    }
+
```

Comment:
what is this for?

Reply:
Would be good to have a tighter unit test instead if that is the case.

Note that in `test_intputs/signals.rs` it has this code

```rust
unsafe extern "C++Qt" {
...
   #[qobject]
   type QTimer;
}
```

Isn't this the same?

Reply:
Hmm, I wonder if that gets parsed differently because it's unsafe? there was one line missing in coverage, I'll remove that block so next CI run we should be able to see if that did affect coverage or if it's now fixed by elsewhere

Reply:
Right this could be the case, but then a unit test tighter could be created rather than another entry in test_inputs